### DISCUSSION TITLE
solution_ray_cast.py is running with single thread

### DISCUSSION THREAD

**Discussion by zzzhello** [Kitware/trame#283] [Trame/VTK]
Dear Trame team,  my computer has intel i7 cpu (6 cpu cores), when i am runing example './trame-tutorial/01_vtk/solution_ray_cast.py' (vtkFixedPointVolumeRayCastMapper is used in this example) , only about 30% cpu is working(not 100% as expected), the raycast rendering is quite slow. Can you give me any suggestions to fix this? thanks in advance.
 
I have tried 'volumeMapper.SetNumberOfThreads(12) ' but not working.

**Comment by jourdain** [comment]
Thanks for reaching out but you should ask your question [on VTK discourse](https://discourse.vtk.org/)
